pycbc/noise/gaussian.py

```diff
@@ -125,5 +122,36 @@ def noise_from_psd(length, delta_t, psd, seed=0):
         SimNoise(segment, stride, psd, randomness)
         
     return noise_ts
-   
-__all__ = ['noise_from_psd', 'frequency_noise_from_psd']
+  
+def noise_from_string(psd_name, length, delta_t, seed=0, low_frequency_cutoff=10.0):
```

Comment:
I might suggest using seed=None as default and then allowing the code to seed using a random seed if seed=None. Otherwise people might run this multiple times (or in multiple instances) and keep getting the same noise.